docs/source/architecture.rst

```diff
@@ -4,7 +4,7 @@ Architecture Reference
 .. toctree::
    :maxdepth: 1
 
-   Hyperledger Fabric CA's User Guide <http://hyperledger-fabric-ca.readthedocs.io/en/latest>
+   Hyperledger Fabric CA's User Guide <http://hyperledger-fabric-ca.readthedocs.io/en/{BRANCH_DOC}>
```

Comment:
This one is an issue, as Fabric-CA no longer keeps up with Fabric. But I will correct in a PR on top of this